source/features/clean-conversation-sidebar.tsx

```diff
@@ -101,6 +101,7 @@ async function clean(): Promise<void> {
 
 	// Linked issues/PRs
 	select('[aria-label="Link issues"] p')!.remove(); // "Successfully merging a pull request may close this issue."
+	select('[aria-label="Link issues"] p')?.remove(); // "Successfully merging a pull request may close this issue." Not shown if issues are disabled.
```

Comment:
There are 2? The use a loop

Reply:
No This issue is really unrelated. I just threw it into the PR.

I can revert if you want

Reply:
Reverted